pkg/tsdb/influxdb/influxdb.go

```diff
@@ -92,7 +92,13 @@ func (e *InfluxDBExecutor) Query(ctx context.Context, dsInfo *models.DataSource,
 		return nil, err
 	}
 
-	httpClient, err := dsInfo.GetHttpClient()
+	var httpClient *http.Client
+	version := dsInfo.JsonData.Get("version").MustString("1x")
+	if version == "2x" {
+		glog.Info("TODO use org & token to create a new client")
```

Comment:
@kylebrandt -- I have asked them to confirm what the proper request structure is.  My understanding was basic auth with user=org and token=password, but that does not seem to work